project/Build.scala

```diff
@@ -773,7 +775,7 @@ object Build {
 
   lazy val linker: MultiScalaProject = MultiScalaProject(
       id = "linker", base = file("linker/jvm")
-  ).zippedSettings("minilib", "library")(
```

Comment:
I believe this overload of `zippedSettings` is now unused and can be removed.

Reply:
Good catch! Removed.